app/dao/notifications_dao.py

```diff
@@ -711,7 +711,8 @@ def dao_get_letters_to_be_printed(print_run_deadline, postage, query_limit=10000
         Notification.notification_type == LETTER_TYPE,
         Notification.status == NOTIFICATION_CREATED,
         Notification.key_type == KEY_TYPE_NORMAL,
-        Notification.postage == postage
+        Notification.postage == postage,
+        Notification.billable_units > 0
```

Comment:
yeah i agree that it's probs quite small chance that this would affect things, and this deffo is the right choice in my eyes. im fine with us putting this check in for now 👍